base-math/extendddtf.c

```diff
@@ -30,14 +30,61 @@
 #define NAME extend
 #endif
 
+#include <float.h>
+
 #include "dfpacc.h"
 #include "convert.h"
+#include "convert_helpers.h"
 
 CONVERT_WRAPPER(
 // extendddtf, trunctdtf
 	volatile double df_part1, df_part2;
+	int exp;
 	SRC_TYPE dd_part1, dd_part2;
 
+	/* Get normalized exponent.  */
+	(void) getmantandexpd128 (a, &exp, 17, 1e17DL);
```

Comment:
Is there a reason why your have to pass `shift = 17` and `_y = 1e17DL`, here?  In other calls to `getmantandexpd128`, such as in `trunctdsf.c` and `trunctddf.c`, the return of `getmantandexpd128` is actually used, so it makes sense to perform the shift.  Here, the shift has no effect, since you discard the return of `getmantandexpd128`.

For that reason, and in order to avoid a future misunderstanding, I think you should convert this line into:

```
(void) getmantandexpd128 (a, &exp, 0, 0);
```

Reply:
@gftg85  No special reason in fact I am working to not call this function at all since there's lot's of stuff that I don't need there. But, I will send that in another PR. For now I already changed to your sugestion

Reply:
Thanks.